apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * An ordered set of progress bubbles.
+ */
+
+import React from 'react';
+import Radium from 'radium';
+import ProgressBubble from './ProgressBubble';
+import color from "@cdo/apps/util/color";
+
+const styles = {
+  main: {
+    position: 'relative',
+    display: 'inline-block'
+  },
+  withBackground: {
+    display: 'inline-block',
+    position: 'relative'
+  },
+  background: {
+    height: 10,
+    backgroundColor: color.lighter_gray,
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    marginTop: (ProgressBubble.height - 10) / 2,
+    marginBottom: (ProgressBubble.height - 10) / 2,
+  },
+  backgroundFirst: {
+    left: 15
+  },
+  backgroundLast: {
+    right: 15
+  },
+  bubble: {
+    position: 'relative',
+  }
+};
+
+const ProgressBubbleSet = React.createClass({
+  propTypes: {
+    startingNumber: React.PropTypes.number.isRequired,
+    statuses: React.PropTypes.arrayOf(
+      React.PropTypes.string
+    ).isRequired,
+    urls: React.PropTypes.arrayOf(
+      React.PropTypes.string
+    ).isRequired
```

Comment:
I'm not too sure how I want this to end up yet. I find our current data difficult to parse because we (a) have so many levels of nesting and (b) end up having a lot of data in there that's unused - at least for this particular component. I could do the same sort of thing where I have a selector that grabs only the pieces I need (url and status right now), which would at least solve b.

I think for the near term, I'm going to stick in a TODO and think about this a little bit more. I think I'll have a better sense of what I want the shape of the data to look like as I start workign on the detail view.